### DISCUSSION TITLE
Making Base Stats Higher Than 255 (TinyINT Restriction)

### DISCUSSION THREAD

**Discussion by krlsmnk** [TrinityCore/TrinityCore#28415] [Q&A] [answered]
Saw in the SQL table "player_levelstats" that the base stats (STR, AGI, etc.) were stored in TINYINT format (0-255). 
I changed the data type to accommodate higher values, and entered some for levels 81-100. 
In game, I noticed the base stats still wrapped around after exceeding 255, so something other than just this table is imposing that restriction. 
Anyone know what I can do to change this?

**Comment by Aokromes** [comment]
it's very likely client limitation.

**Comment by funjoker** [comment]
Level 81-100?
On Shadowlands you are on level 60 max.
So I guess you running Legion or BfA

**Reply by Jinnaix** [reply]
Don't ignore the existence of 335 :'(

**Comment by Shauren** [comment, marked as answer]
Just changing the type in database does not change how its stored in memory when loaded by worldserver, you will have to edit code

In addition, client limit is int16 (smallint), going full int32 (int) wont help you at all
